atd-vzv/src/views/summary/SummaryView.js

```diff
@@ -33,65 +33,77 @@ const SummaryView = () => {
   const [totalCrashes, setTotalCrashes] = useState(null);
 
   useEffect(() => {
-    const fatalitiesUrl = `${crashEndpointUrl}?$where=death_cnt > 0 AND crash_date between '${summaryCurrentYearStartDate}T00:00:00' and '${summaryCurrentYearEndDate}T23:59:59'`;
-    const yearsOfLifeLostUrl = `${personEndpointUrl}?$where=prsn_injry_sev_id = '4' AND crash_date between '${summaryCurrentYearStartDate}T00:00:00' and '${summaryCurrentYearEndDate}T23:59:59'`;
-    const seriousInjuriesUrl = `${crashEndpointUrl}?$where=sus_serious_injry_cnt > 0 AND crash_date between '${summaryCurrentYearStartDate}T00:00:00' and '${summaryCurrentYearEndDate}T23:59:59'`;
-    const totalCrashesUrl = `${crashEndpointUrl}?$where=crash_date between '${summaryCurrentYearStartDate}T00:00:00' and '${summaryCurrentYearEndDate}T23:59:59'&$limit=100000`;
-
-    axios.get(fatalitiesUrl).then(res => {
-      setFatalities(calculateTotalFatalities(res.data));
-    });
-
-    axios.get(yearsOfLifeLostUrl).then(res => {
-      setYearsOfLifeLost(getYearsOfLifeLost(res.data));
-    });
+    const fatalitiesUrl = `${crashEndpointUrl}?$where=(death_cnt > 0 AND crash_date between '${summaryCurrentYearStartDate}T00:00:00' and '${summaryCurrentYearEndDate}T23:59:59') OR (death_cnt > 0 AND crash_date between '${summaryLastYearStartDate}T00:00:00' and '${summaryLastYearEndDate}T23:59:59')`;
```

Comment:
All these URLs now request records for the previous and current year. The totals for each year are now sorted and then calculated in the front-end.

Reply:
Oooff! really good stuff here, the components look amazing.

Reply:
nice! no extra tax on us. make Tyler Tech pay.